packages/cli/test/integration/generators/model.integration.js

```diff
@@ -119,6 +117,21 @@ describe('lb4 model integration', () => {
       basicModelFileChecks(expectedModelFile, expectedIndexFile);
     });
 
+    it('creates "src/models" directory if it does no exist', async () => {
```

Comment:
typo: `does no exist` -> `does not exist`

Reply:
I fixed it as part of rebase.